detection-rules/attachment_any_html_unsolicited.yml

```diff
@@ -12,11 +12,11 @@ source: |
   type.inbound
   and any(attachments, .file_extension in~ ('htm', 'html') or .file_type == "html")
   and (
-    not profile.by_sender().solicited
-    or (
+    (
       profile.by_sender().any_messages_malicious_or_spam
```

Comment:
this doesn't make sense. You will now only flag on senders that have already flagged before.